### ISSUE TITLE
Human Dataset Visualization

### ISSUE THREAD

**Issue by SOMNATH0904** [abhisheks008/ML-Crate#722]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** :  Human Dataset Visualization
</br></br>:red_circle: **Aim** : Here I will convert the human3.6M dataset from txt file to image from for 10 time steps to predict the 11th step. Similarly I will convert the human3.6M dataset from txt file to skeleton gif data.
</br></br>:red_circle: **Dataset** : N/A
</br></br>:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model. Linear Regression, Ada Boost Algorithm, Decision Tree, Random Forest
</br></br>🔴 **Reference Project Folder**: NA

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : Somnath Shaw
- GitHub Profile Link : https://github.com/SOMNATH0904
- Participant ID (If not, then put NA) : NA
- Approach for this Project : 1) Data Cleaning 2) Data Preprocessing 3) Model Training 4) Predictions and Testing
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.) SSOC Contributor

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by abhisheks008** [comment]
Please share the dataset for this problem statement.

**Comment by SOMNATH0904** [comment]
Taken the dataset from this website -> *http://vision.imar.ro/human3.6m/description.php*

**Comment by abhisheks008** [comment]
You are planning to predict the steps of the humans using your models right? If that is the problem statement, then what are the models you are planning to implement here? Mention at least 4-5 models for this.

**Comment by SOMNATH0904** [comment]
Yes, the plan is to predict human steps using models in this project. I am planning to implement several models, including Convolutional Neural Networks (CNNs) for feature extraction, Long Short-Term Memory (LSTM) networks for handling sequential data, and 3D Pose Estimation models like the Human Pose Estimation Network (HPE) for predicting 3D poses from 2D images. Additionally, Generative Adversarial Networks (GANs) will be used to generate realistic human motion sequences, and Temporal Convolutional Networks (TCNs) will be considered for capturing long-range dependencies in sequential data.

**Comment by abhisheks008** [comment]
Assigned @SOMNATH0904

**Comment by abhisheks008** [comment]
One issue at a time @SOMNATH0904

**Comment by SOMNATH0904** [comment]
@abhisheks008 Can you now please assign this to me. I have the project. I want to contribute.

**Comment by abhisheks008** [comment]
Assigned @SOMNATH0904
